Gemfile

```diff
@@ -29,6 +29,7 @@ else
 end
 
 gem 'json'
+gem 'json-schema'
```

Comment:
We already bundle `json_schemer` gem which should be used instead. See: https://github.com/discourse/discourse/pull/11721#issuecomment-760550775

You can consult with our resident json schema expert, @oblakeerickson! 😀